analysis/download_s3_files.ipynb

```diff
@@ -117,11 +118,33 @@
     "outfile_path = Path(f\"{path}/vi-normalized-with-liftover.jsonl.gz\")\n",
     "download_s3(url, outfile_path)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <a id='toc1_3_'></a>[ClinVar processed CNV data](#toc0_)\n",
+    "If you have not already run `prep_clinvar_cnvs.ipynb`, the following cell can be run to generated a list of processed ClinVar cnvs. This file is provided as input to the `query_match_nch_clinvar_cnvs.ipynb` notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path = \"cnvs/cnv_data\"\n",
+    "Path(path).mkdir(exist_ok=True)\n",
+    "\n",
+    "url = f\"{MANUSCRIPT_S3_URL}/clinvar/ClinVar-CNVs-normalized.csv.gzip\"\n",
+    "outfile_path = Path(f\"{path}/ClinVar-CNVs-normalized.csv.gzip\")\n",
```

Comment:
We may want to update the readme saying this file is downloaded in the s3 nb

Reply:
I don't see the `ClinVar-CNVs-normalized.csv.gzip` in the readme

Reply:
This should be updated to `ClinVar-CNVs-normalized.csv.gzip`

Reply:
oh wait. that's not the gzip